src/wallet/rpcwallet.cpp

```diff
@@ -3176,6 +3176,7 @@ Value z_sendmany(const Array& params, bool fHelp)
             "z_sendmany \"fromaddress\" [{\"address\":... ,\"amount\":...},...] ( minconf )\n"
             "\nSend multiple times. Amounts are double-precision floating point numbers."
             "\nChange from a taddr flows to a new taddr address, while change from zaddr returns to itself."
+            "\nWhen sending coinbase utxos to a zaddr, change is not alllowed.  The entire utxo must be consumed."
```

Comment:
(In practice, the behaviour of the built-in miner is to mine each block to a different t-addr, so users won't run into the multiple UTXO case unless they are changed that or are using another miner.)